### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by bennyschirm** [mbrlabs/Lorien#133] [Q&A] [answered]
It's been a while since the last release and there are quite a few new features now. Are you planning to do a new release anytime soon?

**Comment by mbrlabs** [comment]
Yeah, i actually wanted to do a release soon. Maybe even this weekend :)

**Reply by bennyschirm** [reply]
Awesome, can't wait! Although i am using it already (custom build)

**Comment by mbrlabs** [comment, marked as answer]
I just released version 0.5.0. 
You can download it here: https://github.com/mbrlabs/Lorien/releases/tag/v0.5.0

**Reply by bennyschirm** [reply]
Nice, thank you!
